test/src/tests/stats.test.js

```diff
@@ -31,6 +31,10 @@ describe('home page stats', () => {
     const dandisetName = `dandiset${uniqueId()}`;
     const dandisetDescription = `Description! ${uniqueId()}`;
 
+    // The value that `homePage.getStat` looks for is retrieved via XHR, so
+    // wait until all requests are finished before trying to retrieve it.
+    await page.waitForNavigation({ waitUntil: 'networkidle0' });
```

Comment:
This is a really neat feature in puppeteer, but I have a vague memory of some problem with a third party package (google analytics or sentry or something) that was sending requests every few seconds, which prevented `waitForNavigation` from ever finishing. I'm happy to try it again, but be aware if it ever starts acting up.

Reply:
https://github.com/dandi/dandiarchive/pull/848/commits/e8102c3b44c010b145b25affdb1b39552b8702e3 I updated it so it times out at 10 seconds (which should be more then enough time for any of our http requests to complete anyways)